pkg/cluster/kubeclient/backup.go

```diff
@@ -0,0 +1,84 @@
+package kubeclient
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+)
+
+func (u *kubeclient) BackupCluster(ctx context.Context, backupImage, backupName string) error {
+	u.log.Infof("running an etcd backup using %s", backupImage)
+	namespace := "openshift-etcd"
+	bk, err := u.client.BatchV1().Jobs(namespace).Create(&batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      backupName,
+			Namespace: namespace,
+		},
+		Spec: batchv1.JobSpec{
```

Comment:
All of this is already defined in the pre-existing "etcd-backup-hourly" CronJob object.  We shouldn't duplicate.
In pkg/addons/data rename the CronJob to a better name ("etcd-backup").
Fetch the CronJob here using the hard-coded name, and use the JobTemplate to create a Job object.